script/Arcueid/Day 13/13_00_ARC13_1B.txt

```diff
@@ -1086,7 +1089,8 @@ I put strength into my legs.
 [sha:6a605f261f6bb38b34233bad514baa68e77e7a39]{
 -- Page 167, Offset 32122.
 -- 乱れていく呼吸をなんとか誤魔化しながら、ベッドから立ちあがった。
-I somehow move the disordered breathing to the back of my head and stand up from the bed.
+// fuck you 誤魔化しながら
+I somehow move the disordered breathing to the back of my mind and stand up from the bed.
```

Comment:
Hiding my true state, I somehow move my rough breathing to the back of my mind and